seed_data/availabilityData.js

```diff
@@ -0,0 +1,205 @@
+var availabilityData = {
+  name: 'Availability',
```

Comment:
All this data is pretty useless, it should list locations around here... I guess it's `DB` though.